src/steps/owners/index.ts

```diff
@@ -1,31 +1,39 @@
 import {
+  createDirectRelationship,
   IntegrationStep,
   IntegrationStepExecutionContext,
+  RelationshipClass,
 } from '@jupiterone/integration-sdk-core';
 import { createAPIClient } from '../../client';
 import { IntegrationConfig } from '../../config';
 import { getEntityKey } from '../../utils';
 import { Entities, IntegrationSteps, Relationships } from '../constants';
-import { createOwnerEntity, createOwnerRoleRelationship } from './converters';
+import { createUserEntity } from './converters';
 
 export async function fetchOwners({
   jobState,
   instance,
 }: IntegrationStepExecutionContext<IntegrationConfig>) {
   const apiClient = createAPIClient(instance.config);
   await apiClient.iterateOwners(async (owner) => {
-    // Intermediate step to get owner role through user endpoint
     const { roleId } = await apiClient.fetchUser(owner.userId.toString());
 
     const roleEntity = await jobState.findEntity(
       getEntityKey(Entities.ROLE, roleId),
     );
-    const ownerEntity = createOwnerEntity(owner);
-    await jobState.addEntity(ownerEntity);
+    const userEntity = createUserEntity(owner);
+    await jobState.addEntity(userEntity);
 
-    if (roleEntity && ownerEntity) {
+    if (roleEntity && userEntity) {
       await jobState.addRelationship(
-        createOwnerRoleRelationship(ownerEntity, roleEntity),
+        createDirectRelationship({
+          _class: RelationshipClass.ASSIGNED,
+          from: userEntity,
+          to: roleEntity,
+          properties: {
+            _type: Relationships.USER_ASSIGNED_ROLE._type,
+          },
```

Comment:
This isn't necessary, since the type is declared below in relationships array :slightly_smiling_face:

Reply:
Resolved in a11f791